src/main/webapp/dev/DatePickerJsTest.js

```diff
@@ -0,0 +1,38 @@
+QUnit.module('datepicker.js');
+
+QUnit.test('getMinDateForEndDate(startDate)', function(assert) {
+    assert.deepEqual(getMinDateForEndDate(new Date('02/25/2014')), new Date('02/25/2014'), 'Return startDate');
+});
+
+QUnit.test('getMaxDateForStartDate(endDate)', function(assert) {
+    assert.deepEqual(getMaxDateForStartDate(new Date('02/04/2014')), new Date('02/04/2014'), 'Return endDate');
+});
+
+QUnit.test('getMinDateForPublishDate(visibleDate)', function(assert) {
+    assert.deepEqual(getMinDateForPublishDate(new Date('10/13/2014')), new Date('10/13/2014'), 'Return visibleDate');
+});
+
+QUnit.test('getMaxDateForVisibleDate(startDate,publishDate)', function(assert) {
+    assert.deepEqual(getMaxDateForVisibleDate(new Date('08/25/2014'), null), new Date('08/25/2014'),
+            'When publishDate is null, Return minDate = startDate');
+});
+
+QUnit.test('getMaxDateForVisibleDate(startDate,publishDate)', function(assert) {
+    assert.deepEqual(getMaxDateForVisibleDate(new Date('10/13/2014'), new Date('08/25/2014')), new Date('08/25/2014'),
```

Comment:
I would suggest to put every related test case to one big `QUnit.test('getMaxDateForVisibleDate(startDate,publishDate)')` rather than separating them.
e.g.
``` javascript
QUnit.test('getMaxDateForVisibleDate(startDate,publishDate)', function(assert) {
     assert.deepEqual(getMaxDateForVisibleDate(new Date('08/25/2014'), new Date('08/26/2014')), new Date('08/25/2014'), 'startDate is smaller than endDate');
     assert.deepEqual(getMaxDateForVisibleDate(new Date('08/25/2014'), new Date('08/25/2014')), new Date('08/25/2014'), 'startDate equals to endDate');
     //..
 });
```